client/performance-profiler/components/metric-scale/index.tsx

```diff
@@ -14,10 +14,14 @@ export const MetricScale = ( { metricName, value, valuation }: Props ) => {
 		metricsTresholds[ metricName as keyof typeof metricsTresholds ];
 
 	const formatValue = ( value: number ): string => {
+		if ( value === null || value === undefined ) {
+			return '';
+		}
+
 		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( metricName ) ) {
-			return `${ +( value / 1000 ).toFixed( 2 ) }`;
+			return `${ max2Decimals( value / 1000 ) }`;
```

Comment:
Why the `toFixed(2)` doesn't work here?

Reply:
We receive some values as string. I inspected this for a while and `cls` and `fid` sometimes are cast as string. This conversion also needs to be done in multiple places so I moved it to a function.